cityofaustin/atd-moped
# Applies misc DB updates to project funding

## Associated issues
- https://github.com/cityofaustin/atd-data-tech/issues/13113

## Testing
**URL to test:** Local

**Steps to test:**
1. start the hasura cluster and editor
2. visit any project's "Funding" tab.
3. Add a new funding source.
 - Verify that `CapMetro ILA 2018` is listed in the **Source** dropdown menu
 - Verify these options in the **Program** dropdown menu
   - `Large CIP`
   - `Sidewalk Fee in Lieu`
   - `Sidewalk Rehab`
   - `Street Impact Fee`
   - `Street Rehabilitation`
   - `Traffic Mitigation Fees`
   - `Transit Enhancement Program`


---
#### Ship list
